### ISSUE TITLE
Open Slack in the correct team

### ISSUE THREAD

**Issue by BasThomas** [tryswift/trySwiftAppFinal#39]
https://api.slack.com/docs/deep-linking

The Team ID should be provided by the Data pod.

**Comment by funmia** [comment]
@BasThomas Is this still a requested feature? If so I can create a pr for it. The only problem is that I haven't been able to build the app on a device to test properly because of code signing errors.

**Comment by BasThomas** [comment]
This is still valid an up for grabs! You can find the deeplinking documentation [here](https://api.slack.com/docs/deep-linking#open_slack).

The current code / url that is opened in the app can be found [here](https://github.com/tryswift/trySwiftAppFinal/blob/2acf568e337003538c604c9e70b8cd4e66aebd63/trySwift/MoreTableViewController.swift#L207).

Regarding building the app: can you try to change the bundle identifier? You can change it in Project > App Target > General.

Let me know if that doesn't work!

**Comment by funmia** [comment]
Thank you @BasThomas I've made the changes, I just need access to create a pr now. I tried changing the bundle id, for the different targets yesterday but I eventually got stuck on this error;
```error: Embedded binary's bundle identifier is not prefixed with the parent app's bundle identifier.```

I can't change it anymore though, this is what I have in Xcode now. I tried deleting old provisioning profiles but that doesn't help.

<img width="611" alt="screen shot 2018-09-11 at 07 44 21" src="https://user-images.githubusercontent.com/22508999/45342832-072ed900-b597-11e8-99b5-dca3237c98ae.png">
